gapic/src/main/com/google/gapid/perfetto/views/FrameEventsSelectionView.java

```diff
@@ -74,36 +71,23 @@ private void setSingleSliceView(State state, FrameEventsTrack.Slices slice) {
     createLabel(main, "Duration:");
     createLabel(main, timeToString(slice.durs.get(0)));
 
-    if (slice.frameStats.get(0) != null) {
       // If the selected event is a displayed frame slice, show the frame stats
-      Composite stats = withLayoutData(createComposite(this, new GridLayout(2, false)),
-          withIndents(new GridData(SWT.LEFT, SWT.TOP, false, false), PANEL_INDENT, 0));
-      withLayoutData(createBoldLabel(stats, "Frame Stats:"),
-          withSpans(new GridData(), 2, 1));
-
-      slice.frameStats.get(0).forEach((k, v) -> {
-        withLayoutData(createBoldLabel(stats, k.toString()),
-            withSpans(new GridData(), 2, 1));
-
-        createLabel(stats, "Frame number: ");
-        createLabel(stats, Long.toString(v.frameNumber));
-
-        createLabel(stats, "Queue to Acquire: ");
-        createLabel(stats, timeToString(v.queueToAcquireTime));
-
-        createLabel(stats, "Acquire to Latch: ");
-        createLabel(stats, timeToString(v.acquireToLatchTime));
-
-        createLabel(stats, "Latch to Present: ");
-        createLabel(stats, timeToString(v.latchToPresentTime));
-      });
-    } else {
-      // Show the frame number associated with the event
-      createLabel(main, "Frame Number:");
-      createLabel(main, Arrays.stream(slice.frameNumbers.get(0))
-          .map(l -> Long.toString(l))
-          .collect(Collectors.joining(", ")));
-    }
+    Composite stats = withLayoutData(createComposite(this, new GridLayout(2, false)),
+        withIndents(new GridData(SWT.LEFT, SWT.TOP, false, false), PANEL_INDENT, 0));
+    withLayoutData(createBoldLabel(stats, "Frame Stats:"),
+        withSpans(new GridData(), 2, 1));
+
+    createLabel(stats, "Frame number: ");
+    createLabel(stats, Long.toString(slice.frameNumbers.get(0)));
+
+    createLabel(stats, "Queue to Acquire: ");
```

Comment:
Maybe also add descriptive string like Display, Application, Wait for GPU, Composition here, to help ramp up the users and let them know the track's meaning?

Reply:
The stats do not exactly completely to the 4 tracks that we are showing. There is some overlap, like queue to acquire and wait for GPU while the latch delay is not covered in the tracks. These stats are meant for those who know about the instant events and don't want to expand the buffers track fully to check on them.